Resources/Prototypes/Entities/Objects/Fun/water_balloons.yml

```diff
@@ -0,0 +1,107 @@
+- type: entity
+  parent: BaseItem
+  id: EmptyWaterBalloon
+  suffix: Empty
+  name: empty water balloon
+  description: Fill it up to reveal it's true color!
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/water_balloons.rsi
+    layers:
+    - state: icon
+  - type: WaterBalloon
+  - type: Item
+    size: Tiny
+  - type: SolutionContainerManager
+    solutions:
+      balloon:
+        maxVol: 10
+  - type: DrawableSolution
+    solution: balloon
+  - type: RefillableSolution
+    solution: balloon
+  - type: DrainableSolution
+    solution: balloon
+  - type: ExaminableSolution
+    solution: balloon
+  - type: Spillable
+    solution: balloon
+
+- type: entity
+  parent: BaseItem
+  id: FilledWaterBalloon
+  suffix: do not spawn :c 
+  name: water balloon
+  description: ready to splash your enemies!
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/water_balloons.rsi
+    layers:
+      - state: icon_inflated_fill-1
+        map: ["enum.SolutionContainerLayers.Fill"]
+      - state: icon_inflated
+        map: [ "spriteMap" ]
+  - type: RandomSprite
+    available:
+      - spriteMap:
+          icon_inflated: Balloons
+  - type: Item
+    size: Normal
+  - type: SolutionContainerManager
+    solutions:
+      balloon:
+        maxVol: 10
+  - type: ExaminableSolution
+    solution: balloon
+  - type: Spillable
+    solution: balloon
+  - type: Appearance
+  - type: SolutionContainerVisuals
+    maxFillLevels: 1
+    fillBaseName: icon_inflated_fill-
+    solutionName: balloon
+  - type: DamageOnLand
+    damage:
+      types:
+        Blunt: 5 
+  - type: Damageable
+    damageContainer: Inorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 5
+      behaviors:
+      - !type:SpillBehavior 
+        solution: food
```

Comment:
```suggestion
        solution: balloon
```

Reply:
im surprised i didnt notice that lol, good catch